python/pyarrow/tests/test_types.py

```diff
@@ -252,6 +255,121 @@ def test_is_primitive():
     assert not types.is_primitive(pa.list_(pa.int32()))
 
 
+@pytest.mark.parametrize(('tz', 'expected'), [
+    (pytz.utc, 'UTC'),
+    (pytz.timezone('Europe/Paris'), 'Europe/Paris'),
+    (pytz.FixedOffset(180), '+03:00'),
+    (datetime.timezone.utc, '+00:00'),
+    (datetime.timezone(datetime.timedelta(hours=1, minutes=30)), '+01:30')
+])
+def test_tzinfo_to_string(tz, expected):
+    assert pa.lib.tzinfo_to_string(tz) == expected
+
+
+@pytest.mark.skipif(sys.version_info <= (3, 7), reason=(
+    "Since python 3.7 the UTC offset for datetime.timezone is not restricted "
```

Comment:
Based on this message, I don't understand why you skip for `sys.version_info <= (3, 7)` rather than `sys.version_info >= (3, 7)`.

Reply:
Removed the skip since we validate ourself that the timedelta has whole number of minutes.